rfcs/0098-community-team.md

```diff
@@ -0,0 +1,190 @@
+---
+feature: community-team
+start-date: 2021-07-31
+author: Irenes (Irene Knapp)
+co-authors: ashkitten
+shepherd-team: (names, to be nominated and accepted by RFC steering committee)
+shepherd-leader: (name to be appointed by RFC steering committee)
+related-issues: (will contain links to implementation PRs)
+---
+
+# Summary
+[summary]: #summary
+
+Establish an official community team to make moderation decisions, model social norms, and mediate interpersonal problems.
+
+# Motivation
+[motivation]: #motivation
+
+There is not currently any official mechanism for moderation action. It's not sustainable to have to call on Graham any time there's a spammer or conflict that requires moderation, and we'd like to help the community become more self-regulating.
+
+# Goals
+[goals]: #goals
+
+* Model and enforce social norms
+* Mediate disputes when doing so is beneficial to the community
+* Respect the privacy of community members
```

Comment:
"Respect" here means that the community team needs to be self-critical and regard itself as a potential threat to privacy. As a privacy professional I think it's incredibly important to always keep oneself in mind as a potential threat to others, and to treat that responsibility seriously. Otherwise you run into the problem where everybody who violates people's privacy believes that they do so with moral authority.

Doxxing would almost certainly fall under harassment, which would violate the "empathy" value among others. So it's already in scope, separate from the privacy goal.

Reply:
Wait, I thought doxxing was mostly against inclusion, no?